src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -150,6 +153,31 @@ static int client_receive_message_udp(
                 void *userdata);
 static void client_stop(sd_dhcp_client *client, int error);
 
+int sd_dhcp_client_id_to_string(char **ret, const void *data, size_t len) {
+        const sd_dhcp_client_id *clientid = data;
```

Comment:
Just a nit-pick. Shouldn't we call this one client_id or it is deliberate to call it clientid?

Reply:
not a biggie, some other places use clientid variables. but anyway, I just renamed.